envergo/templates/haie/moulinette/result/soumis.html

```diff
@@ -10,11 +10,11 @@ <h6>
 </p>
 
 <h6>
-  🌳 <strong>Plantation requise&nbsp;: {{ length_to_remove }}&nbsp;m</strong>
+  🌳 <strong>Plantation requise&nbsp;: {{ minimum_length_to_plant }}&nbsp;m</strong>
```

Comment:
On n'utilise les entités html nulle part dans les templates. Ça me semble plus cohérent de ne pas le faire ici non plus.

```suggestion
  🌳 <strong>Plantation requise : {{ minimum_length_to_plant }} m</strong>
```